voice_insights_events_test.go

```diff
@@ -30,4 +30,6 @@ func TestGetCallEventsPage(t *testing.T) {
 	if page.Events[0].Edge != "client_edge" {
 		t.Errorf("expected Edge to be 'client_edge', got %s", page.Events[0].Edge)
 	}
+
+	t.Cleanup(s.Close)
```

Comment:
Ah... if anything in the test panics, we won't run the cleanup if it's declared at the end, so it's ideal to put this up where we create `s`. Basically I think use it like a defer.